extensions/amp-truncate-text/1.0/test/test-amp-truncate-text.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../amp-truncate-text';
+import {toggleExperiment} from '../../../../src/experiments';
+import {waitFor} from '../../../../testing/test-helper';
+
+describes.realWin(
+  'amp-truncate-text-v1.0',
+  {
+    amp: {
+      extensions: ['amp-truncate-text:1.0'],
+    },
+  },
+  (env) => {
+    let win, doc, element;
+
+    beforeEach(async function () {
+      win = env.win;
+      doc = win.document;
+      toggleExperiment(win, 'bento-truncate-text', true, true);
+    });
+
+    it('example test renders', async () => {
+      element = doc.createElement('amp-truncate-text');
```

Comment:
TODO: Replicate the existing `amp-truncate-text` tests from 0.1 to 1.0 here.

Reply:
Out of this PR's scope, but this continues happening and there might be value in consolidating common 0.1/1.0 AMP-level tests at some point.

Reply:
I've had that thought too; ideally, since we want the APIs to be as close as possible, the same tests should work for both versions of the component. If you think it's worthwhile and file a tracking issue, I'll look into tweaking the test framework to allow shared tests for multiple versions

Reply:
^^^ I like this approach w.r.t. e2e tests only

Reply:
See https://github.com/ampproject/amphtml/pull/32176